File: pi/src/model/sensordata.py
from datetime import datetime


class SensorData:
    def __init__(self):
        self.temperature = 0
        self.humidity = 0
        self.totalConsumption = 0
        self.momentaryConsumption = 0
        self.currentDate = datetime.now()
        self.light = 0
        self.soilMoisture = 0

        self.heaterStatus = False
        self.lightStatus = False
        self.humidifierStatus = False
        self.waterSysStatus = False

File: server/src/src/main/java/hu/ekke/smartgardenserver/model/SensorData.java
package hu.ekke.smartgardenserver.model;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Data
public class SensorData {
    private Float temperature;
    private Float humidity;
    private Float totalConsumption;
    private Float momentaryConsumption;
    private String currentDate;
    private Float light;
    private Float soilMoisture;
    private Boolean heaterStatus;
    private Boolean lightStatus;
    private Boolean humidifierStatus;
    private Boolean waterSysStatus;

}
